cmd/notary/main.go

```diff
@@ -9,6 +9,7 @@ import (
 	"os/signal"
 
 	"github.com/canonical/notary/internal/config"
+	"github.com/canonical/notary/internal/logger"
```

Comment:
I think it looks a big weird to use `l` in this file for logging calls. We could alias the import instead, and at least the logging calls would look like the other files.

```suggestion
	l "github.com/canonical/notary/internal/logger"
```

Reply:
Ok sure, Done!